io.github.beamerpresenter.beamerpresenter.yml

```diff
@@ -0,0 +1,65 @@
+app-id: io.github.beamerpresenter.beamerpresenter
+branch: master
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: beamerpresenter
+build-options:
+  strip: true
+  no-debuginfo: true
+finish-args:
+  - "--socket=x11"
+  - "--socket=wayland"
+  - "--socket=pulseaudio"
+  - "--filesystem=xdg-documents"
+  - "--filesystem=xdg-download"
+  - "--filesystem=xdg-public-share"
+  - "--filesystem=xdg-desktop"
+  - "--filesystem=xdg-config/beamerpresenter"
+  - "--own-name=io.github.beamerpresenter.beamerpresenter"
```

Comment:
the app-id which is a valid d-bus name is granted by default, you don't need this.

```suggestion
```